### PR TITLE
BIOWDL-524: Add parameter_meta outputs.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md.
- [x] `parameter_meta` for each task is up to date.

I have added `outputs` to tasks that already had a `parameter_meta` section.
Since I don't have experience with a lot of these tools, I would like to ask for reviewers to
double check the descriptions of tasks they have used previously.
Let me know what I can and should improve!

This is a continuation of: https://github.com/biowdl/tasks/pull/261

### CODE REVIEW

**Comment by JasperBoom** [comment]
Hey reviewers, besides checking the outputs I have added and described. I could use
some help with filling the empty output in Picard and somaticseq.
Could you guys maybe suggest or add the outputs for those remaining tools?
I don't have much or no experience with running them. And the manuals aren't
helping very much.

**Comment by JasperBoom** [comment]
Secondly, I have added a file called TO-DO.md. I will take this out before
merging I suppose. But it shows the files/tasks that we haven't updated to the
new SLURM standard.
It also shows a few files that import other files, do we still use
those imports?

**Comment by rhpvorderman** [comment]
Miniwdl has a check function that checks for imports that are not used. You could run that and see if things show up.
The unfinished tasks are not a problem. This is already a lot of progress. We can fix those tasks later when it turns out they cause problems. Our CI checks for any used tasks that are not compliant, if I am correct.

**Review by rhpvorderman — APPROVED** [review]

**Comment by JasperBoom** [comment]
> Miniwdl has a check function that checks for imports that are not used. You could run that and see if things show up.
> The unfinished tasks are not a problem. This is already a lot of progress. We can fix those tasks later when it turns out they cause problems. Our CI checks for any _used_ tasks that are not compliant, if I am correct. Tasks that are not used are less important.

Running the miniwdl check is a good idea, thanks! I will do that.

File: TO-DO.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#TO DO
```
Comment (cagaser) [inline TO-DO.md:1]: Convenient to have

File: biopet/bamstats.wdl [hunk 2]
```diff
@@ -24,18 +24,19 @@ import "../common.wdl" as common
 
 task Generate {
```
Comment (cagaser) [inline biopet/bamstats.wdl:25]: Missing parameters

File: biopet/biopet.wdl [hunk 3]
```diff
@@ -24,15 +24,16 @@ import "../common.wdl"
```
Comment (cagaser) [inline biopet/biopet.wdl:24]: Missing some parameters meta, but we are going to remove this anyways right?

File: gffcompare.wdl [hunk 4]
```diff
@@ -142,6 +142,15 @@ task GffCompare {
         namePrefix: {description: "Equivalent to gffcompare's `-p` option.", category: "advanced"}
         timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
         dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.", category: "advanced"}
+
+        # outputs
+        annotated: {description: "Annotated GTF file."}
+        loci: {description: ""}
```
Comment (cagaser) [inline gffcompare.wdl:148]: empty description

File: biopet/biopet.wdl (hunk 3 above)
Comment (JasperBoom) [reply biopet/biopet.wdl:24]: Indeed, the biopet folder will be removed. I have not added parameter_meta to tasks that have not been updated to the SLURM standard. So if the tasks doesn't have time_minutes and in some cases DockerImage.

File: gffcompare.wdl (hunk 4 above)
Comment (JasperBoom) [reply gffcompare.wdl]: Good eye, I've update it.

**Pull request merged**